q7z/q7z_facade.cpp

```diff
@@ -344,7 +344,7 @@ static quint32 getUInt32Property(IInArchive *archive, int index, int propId, qui
 static QFile::Permissions getPermissions(IInArchive *archive, int index, bool *hasPermissions)
 {
     quint32 attributes = getUInt32Property(archive, index, kpidAttrib, 0);
-    QFile::Permissions permissions = nullptr;
+    QFile::Permissions permissions = static_cast<QFile::Permissions>(0);
```

Comment:
This is fine, but I would prefer the default `QFlags` class constructor: `QFile::Permissions()`